test/test_treetensornetworks/test_solvers/test_contract.jl

```diff
@@ -25,6 +25,14 @@ using Test
   Hpsi = apply(H, psi; alg="fit", init=psi')
   @test inner(psi, Hpsi) ≈ inner(psi', H, psi) atol = 1E-5
 
+  # Test basic usage for use with multiple ProjTTNApply with default parameters
+  # BLAS.axpy-like test
+  os_id = OpSum()
+  os_id += -1, "Id", 1, "Id", 2
+  minus_identity = mpo(os_id, s)
+  Hpsi = apply([H, minus_identity], [psi, copy(psi)]; alg="fit", init=psi')
```

Comment:
I don't think `apply` is the right thing to call this, I was pretty confused about what this meant.

For an API for summing a product of networks, I think it makes more sense to call this:
```julia
sum([(H, psi), (minus_identity, psi)]; kwargs...)
```
i.e. group by which networks are being contracted.

For the `apply` version, I don't think an API is obvious. An experimental thing I was trying out in `ITensors.jl` was using upper case function names to represent lazy function applications, so:
```julia
sum([Apply(H, psi), Apply(minus_identity, psi)]; kwargs...)
```
Otherwise we could go with:
```julia
sum_apply([(H, psi), (minus_identity, psi)]; kwargs...)
```
i.e. denoting it is a sum of applications of operators onto states.

Reply:
I went with the last option
```
sum_apply([(H, psi), (minus_identity, psi)]; kwargs...)
```